### ISSUE TITLE
cant see errors during ./nodebb build

### ISSUE THREAD

**Issue by barisusakli** [NodeBB/NodeBB#5913]
If an error occurs during template compiling no errors are shown.

```
5/9 12:18:48 [9487] - info: [build] Building in parallel mode
5/9 12:18:48 [9487] - info: [build]         plugin static dirs  build started
5/9 12:18:48 [9487] - info: [build]          requirejs modules  build started
5/9 12:18:48 [9487] - info: [build]           client js bundle  build started
5/9 12:18:48 [9487] - info: [build]            admin js bundle  build started
5/9 12:18:48 [9487] - info: [build]         client side styles  build started
5/9 12:18:48 [9487] - info: [build] admin control panel styles  build started
5/9 12:18:48 [9487] - info: [build]                  templates  build started
5/9 12:18:48 [9487] - info: [build]                  languages  build started
5/9 12:18:48 [9487] - info: [build]                     sounds  build started
5/9 12:18:56 [9487] - info: [build]            admin js bundle  build completed in 8.474sec
5/9 12:19:05 [9487] - info: [build]           client js bundle  build completed in 17.707sec
5/9 12:19:05 [9487] - info: [build]                     sounds  build completed in 17.675sec
5/9 12:19:06 [9487] - info: [build]         plugin static dirs  build completed in 18.122sec
baris@foobar:~/nodebb$

**Comment by RoiEXLab** [comment]
Yeah, that was happening to me too in #5789

**Comment by pitaj** [comment]
The only place where an actual error isn't caught is in the JS minifying phase as far as I can tell. Unfortunately it's difficult to catch errors related to memory limits.

**Comment by julianlam** [comment]
Closed via #5916
